src/WebJobs.Script.WebHost/BreakingChangeAnalysis/ChangeAnalysisService.cs

```diff
@@ -128,6 +129,8 @@ public IEnumerable<AssemblyReport> LogBreakingChangeReport(CancellationToken can
             cancellationToken.ThrowIfCancellationRequested();
 
             var generator = new BreakingChangeReportGenerator();
+            var removedAssembliesString = FileUtility.ReadResourceString($"{ScriptConstants.ResourcePath}.AssembliesRemoved.txt");
+            var removedAssemblies = removedAssembliesString.Split("\r\n").ToList();
```

Comment:
You can use a `StringReader` and be more efficient/simpler here.

Reply:
I am digging more into logs - we probably do not need this location log. Will sync up offline.

Reply:
done